apps/hpl/2.1/metadata.yml

```diff
@@ -46,48 +48,29 @@
     - mpi/openmpi
     - libs/atlas
 :sources:
-  - 4node_hpl.alces.dat
+  - 8proc_hpl.alces.dat
+  - 8proc_hpl_example.sh.tpl
+  - 8proc_hpl_example.sh.md
 :compile: |
   export TOPdir=`pwd`
   make arch=<%= compiler[:build_name]%> <%= redirect :make %>
 :install: |
   (cd bin && ln -snf <%=compiler[:build_name]%>/* .) <%= redirect(:install) %>
   cp -pav . <%= dest_dir %> <%= redirect(:install) %>
-  mkdir -v <%= dest_dir %>/examples <%= redirect(:install) %>
-  cp -v <%= source('4node_hpl.alces.dat') %> <%= dest_dir %>/examples <%= redirect(:install) %>
+  mkdir -pv <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
+  mkdir -pv <%= dest_dir %>/share/clusterware/guides <%= redirect(:install) %>
+  cp -v <%= source('8proc_hpl.alces.dat') %> <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
```

Comment:
I'd suggest keeping the data separately from the template files -- perhaps in `<%= dest_dir %>/examples`?